### PR TITLE
games-engines/openmw: enable lua5-1, lua5-3, lua5-4

### PR DESCRIPTION
**Pull request by DarthGandalf**
But only on arches which don't support luajit

@darkbasic

### CODE REVIEW

File: games-engines/openmw/openmw-0.48.0.ebuild [hunk 1]
```diff
@@ -137,6 +135,11 @@ src_compile() {
 }
 
 src_test() {
+	# Lua 5.x is supported in theory, but don't work as well, the test fails
+	# Upstream recommends luajit, but it has less arch coverage
+	if [[ ${ELUA} != luajit ]]; then
+		return
+	fi
```
Comment (juippis) [inline games-engines/openmw/openmw-0.48.0.ebuild:142]: I think some `elog` or `ewarn` should be used here to inform tests have been skipped.

**Review by juippis — APPROVED** [review]
Sorry but could you rebase? This doesn't apply cleanly. LGTM otherwise.

File: games-engines/openmw/openmw-0.48.0.ebuild (hunk 1 above)
Comment (DarthGandalf) [reply games-engines/openmw/openmw-0.48.0.ebuild]: Done

**Review by juippis — APPROVED** [review]
Thanks!

**Pull request closed without merging**